src/content/changes/highlights/2023-03-23.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-03-23T11:00:00
+title: Highlights for the week ending March 23, 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+### Observability
+
+- Prometheus-agent: New cluster releases contain a new component which enables us to monitor your cluster and provides the best quality of service. Some cluster upgrades might require manual intervention from our side, especially if you are using Prometheus Operator. Feel free to ask our team for any questions or help needed (@support-atlas). In case you want to know more details about why we introduced such a change please read our [documentation](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
+- Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/)
```

Comment:
```suggestion
- Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/)
```
Link was broken. @TheoBrigitte Is there a grafana release that can we link here?